### ISSUE TITLE
Unstable training

### ISSUE THREAD

**Issue by myt889** [clovaai/rexnet#23]
Hi，
     thanks for your great work.Now I used the rexnet as my backbone for my own classification task,I found that the backbone is not stable(maybe with the maximum 1% of val_acc).I used the Adam optimizer and the ExponentialLR as the lr scheduler, without the warmup strategy. I wond konw that how to make stable training,can you give me some guidence?  Looking forward to your reply.

**Comment by dyhan0920** [comment]
Thank you for your interest in our backbone. In my experience, ReXNet has similar training behavior to other renowned architectures. I think the instability may stem from the training setup; please refer to the [training setting](https://github.com/clovaai/rexnet#training-own-rexnet) here.  

Adam and AdamW both are quite stable optimizers and may not cause any instability in my experience. Presumably, high LR or WD cause the problem, if there is no LR warming up.

**Comment by myt889** [comment]
Thank you for your kindness, I take the Adam optimizer with lr=0.001,WD=1e-4, and make lr scheduler  each epoch by  ExponentialLR with gamma=0.9.

**Comment by dyhan0920** [comment]
@myt889 The setting you used may not cause instability in my experience. Presumably, dataset issues (e.g., labeling noise) affect the instability, and large LR without any LR warming-up harm the training stability as well.
